provider/resource_playbook.go

```diff
@@ -494,27 +618,42 @@ func resourcePlaybookUpdate(data *schema.ResourceData, meta interface{}) error {
 		playbookFailMsg := fmt.Sprintf("ERROR [ansible-playbook]: couldn't run ansible-playbook\n%s! "+
 			"There may be an error within your playbook.\n%v",
 			playbook,
-			runAnsiblePlayErr,
+			runAnsiblePlayOut,
 		)
```

Comment:
`runAnsiblePlayOut` is a byte slice so it should be converted to a string. `%v` right now is just showing the default representation of a byte slice, which looks like an array of integers. I would just replace this whole `Sprintf` thing with something like:
```go
playbookFailMsg := string(runAnsiblePlayOut)
```

Reply:
fixed